.github/workflows/ci-test.yaml

```diff
@@ -297,3 +297,26 @@ jobs:
         env:
           TEST_NODE_BINARY: "${{ github.workspace }}/tests/untracked/oasis-node"
           TEST_RUNTIME_LOADER: "${{ github.workspace}}/tests/untracked/oasis-core-runtime-loader"
+
+  jest-ts-web-core:
+    name: jest-ts-web-core
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js LTS
+        uses: actions/setup-node@v2.1.4
+        with:
+          node-version: "14.x"
```

Comment:
strictly speaking, the quotes are unnecessary

Reply:
I followed the exact template of the other e2e test invocations in this file - should they all be fixed in this PR ?

Reply:
nah. thanks for offering, though!

Reply:
it's "I copied from an earlier test" all the way down. I wonder if it originates from a specimen that didn't have the ".x" and would be parsed as a number